specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-12-01-preview/apimanagement.json

```diff
@@ -542,6 +542,44 @@
       },
       "x-ms-parameter-location": "method"
     },
+    "IdParameter": {
```

Comment:
lets rename this to `PolicyIdFragmentParameter`

Reply:
It was PolicyFragmentIdParameter before, but Maxim asked for this to be generalised so that we can reuse it in the future. 

Most endpoints in APIM have the same ID validation and are already using the new regex validation on the backend but not in this file. This basically introduces a generic ID that can be reused across APIM later on in a PR that fixes the mismatch between this file and the actual implementation.